src/main/scala/dpla/ingestion3/enrichments/VocabEnforcer.scala

```diff
@@ -98,3 +100,26 @@ object DcmiTypeMapper extends VocabEnforcer[String] {
 
   val mapDcmiType: (String) => Any = mapVocab(_, DcmiTypeMap)
 }
+
+/**
+  * Reads in ISO-693-3 data from text file and generates a
+  * map of abbreviations >> normalized names.
+  *
+  */
+object LexvoEnforcer extends VocabEnforcer[String] {
+  val iso_639_3 = ISO_639_3()
+
+  val lexvoStrings = {
+    // TODO make this a config property
+    val bufferedSource = Source.fromFile("./data/iso-639-3/iso-639-3.tab")
```

Comment:
We might need to figure out another way to get at this data since that file will end up in a JAR at some point. It's something like `getClass().getResource("somepath")`, but we can figure it out later.